django/project/models.py

```diff
@@ -970,7 +973,7 @@ def to_representation(self):
             'phase': self.phase,
             'open_source_frontier_tech': self.open_source_frontier_tech,
             'learning_investment': self.learning_investment,
-            'portfolios': list(portfolios),
+            'portfolios': portfolios,
```

Comment:
Are we sure that list(portfolios) is not needed?

Reply:
hmm let's keep it jic, even though i didnt see this function being called somewhere